src/app/forms/components/defect/defect.component.html

```diff
@@ -83,8 +83,17 @@
       <app-text-input formControlName="notes" label="Notes"></app-text-input>
     </ng-container>
 
-    <app-text-input formControlName="prs" label="PRS"></app-text-input>
     <app-radio-group
+      name="prs"
+      formControlName="prs"
+      label="PRS"
+      [options]="[
```

Comment:
This is a very chuncky object to go into the HTML, maybe consider having a getter in the component for it?
```typescript
get options(): Option[] {
  return [
    { value: true, label: 'Yes' },
    { value: false, label: 'No' }
  ];
}
```

Reply:
Alright, thanks for trying.